guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -26,11 +26,11 @@ endif::[]
 # postgresql-setup initdb
 ----
 +
-. Edit the `/var/lib/pgsql/data/postgresql.conf` file:
+. Edit the `{postgresql-conf-dir}/postgresql.conf` file:
 +
 [options="nowrap" subs="verbatim,quotes"]
 ----
-# vi /var/lib/pgsql/data/postgresql.conf
+# vi {postgresql-conf-dir}/postgresql.conf
```

Comment:
this doesn't get replaced, you need those magic options to the codeblock :roll_eyes: